web/packages/teleterm/src/ui/Search/pickers/ActionPicker.tsx

```diff
@@ -126,7 +121,7 @@ export function ActionPicker(props: { input: ReactElement }) {
       return (
         <FilterButton
           key="resource-type"
```

Comment:
Keys need to be made unique, otherwise there's an error about them being the same and removing them is buggy.

Reply:
Right, I forgot about this place.